modules/core/src/main/java/org/apache/ignite/internal/managers/communication/TransmissionHandler.java

```diff
@@ -40,6 +40,15 @@
  * {@link TransmissionPolicy#CHUNK} the <em>#chunkHandler()</em> will be picked up.
  */
 public interface TransmissionHandler {
+    /**
+     * The end of the handled transmission. This means that all resources associated with previously opened
+     * session is freed and can be reused. Generally, it means that transmission topic from now can
+     * safely accept new files.
+     *
+     * @param rmtNodeId Remote node id from which the source request comes from.
+     */
+    public void onEnd(UUID rmtNodeId);
```

Comment:
As I can see. this method is not mandatory - maybe it’s worth adding a default implementation?